packages/core/package.json

```diff
@@ -69,6 +69,7 @@
     "lru-cache": "^10.0.0",
     "nanoid": "^4.0.0",
     "oidc-provider": "^8.2.2",
+    "otplib": "^12.0.1",
```

Comment:
<img width="192" alt="image" src="https://github.com/logto-io/logto/assets/14722250/0cfbb343-274a-4fb3-a636-b3284d0685fe">

do we have a better option?

Reply:
It has 300k download per week, I think that is solid enough, maybe the protocal is stable so the lib does not need to update in recent years.